src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocCommentsTokenTypes.java

```diff
@@ -78,6 +78,24 @@ public final class JavadocCommentsTokenTypes {
 
     /**
      * Plain text content within a Javadoc comment.
+     *
+     * <p>This node represents any plain text that appears in a Javadoc comment,
+     * including spaces and punctuation.</p>
+     *
+     * <p><b>Example:</b></p>
+     * <pre>{@code * This is plain text content.}</pre>
+     *
+     * <b>Tree:</b>
+     * <pre>{@code
+     * JAVADOC_CONTENT -> JAVADOC_CONTENT
+     *     |--NEWLINE -> \r\n
+     *     |--LEADING_ASTERISK ->  *
```

Comment:
please remove nodes above LEADING_ASTERISK, they arenot visible in what you have at `{@code * This is plain text content.}`